tc/autotuner/autotuner-inl.h

```diff
@@ -217,13 +217,7 @@ void TuningHarness<Backend>::doEvaluate(
     printer.record(prof);
     pConf->runtime = prof;
 
-    optionsCache_->recordRuntime(
-        lang::canonicalTc(tcTree_),
-        makeTensorInfoVector(inputs),
-        makeTensorInfoVector(outputs),
-        Backend::backendString(),
-        options,
-        prof);
+    optionsCache_->recordRuntime(*pExecutor, prof);
```

Comment:
Nice, but aren't there many other instances that could be simplified?
Otherwise, it may not be worth the effort.